src/widgets/reflow.rs

```diff
@@ -196,14 +199,18 @@ mod test {
 
     enum Composer {
         WordWrapper,
+        WordWrapperNoTrim,
```

Comment:
We could maybe have `WordWrapper { trim: bool }` instead ?

Reply:
For sure! I was originally looking for the minimal number of changes to the other tests, but this is probably the cleaner approach :)

Reply:
I've gone with an `enum` variant for the moment, but let me know if you think this would be better done with `structs`!

Reply:
I agree with keeping an enum variant here but we could maybe go for an anonymous struct so it is more obvious what the boolean flag is for, e.g.:
```rust
enum Composer {
  WordWrapper { trim: bool },
  LineTruncator
}
```

Reply:
That's an excellent idea, sorry for missing that! I forgot the you could have Enum structs like that! I'll make that change now :)